packages/vats/test/vat-bank.test.js

```diff
@@ -58,10 +63,10 @@ test('communication', async t => {
 
   const zone = makeDurableZone(baggage);
 
-  /** @type {undefined | ERef<import('../src/types.js').BridgeHandler>} */
+  /** @type {undefined | Remote<BridgeHandler>} */
```

Comment:
Drive by learning - 

I see `Remote` is a _type that doesn't assume its parameter is local, but is satisfied with both local and remote references._

For `ERef` - _A reference of some kind for to an object of type T. It may be a direct reference to a local T. It may be a local presence for a remote T. It may be a promise for a local or remote T. Or it may even be a thenable (a promise-like non-promise with a "then" method) for a T._

This reads as roughly the same to me. The `M.eref()` matcher maybe gives more clues - it says the value must be `Passable`. And Passables must be acyclic data that can be marshalled. Is that the difference?

Reply:
check out https://github.com/Agoric/agoric-sdk/pull/9406#pullrequestreview-2074797682 . I'll try to add tests there to illustrate